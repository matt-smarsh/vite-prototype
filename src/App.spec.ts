import { enableAutoDestroy, shallowMount } from "@vue/test-utils"
import Component from "./App.vue"

enableAutoDestroy(afterEach)

it("mounts", async () => {
	const wrapper = shallowMount(Component)
	expect(wrapper.html()).toMatchSnapshot()
})
