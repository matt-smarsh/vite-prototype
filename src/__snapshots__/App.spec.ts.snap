// Vitest Snapshot v1

exports[`mounts 1`] = `
"<div id=\\"app\\"><img alt=\\"Vue logo\\" src=\\"/src/assets/logo.png?import\\">
  <helloworld-stub msg=\\"Hello Vue 2 + TypeScript + Vite\\"></helloworld-stub>
</div>"
`;
